django/conf/global_settings.py

```diff
@@ -310,6 +310,28 @@
 # file system instead of into memory.
 FILE_UPLOAD_MAX_MEMORY_SIZE = 2621440  # i.e. 2.5 MB
 
+# Maximum size in bytes of request data that will be read before raising a
+# SuspiciousOperation. This applies to any data in form data requests,
+# excluding file uploads. Since web servers typically do not do deep
```

Comment:
Let's move everything after "Since...." for these settings into the documentation.